code/modules/chemistry/Reagents-FoodDrink.dm

```diff
@@ -119,6 +119,82 @@ datum
+
+			on_mob_life(var/mob/target, var/mult = 1)
+				if(!target) target = holder.my_atom
+				var/ld_amt = holder.get_reagent_amount(src.id)
+				var/mob/living/carbon/human/M = target
+				if(ld_amt > 40)
+					regret = TRUE //If you go too far, you'll wish you hadn't
+				if (!istype(M))
+					return
+				var/coke = pick("conk", "croke", "ckoe", "clock", "coge", "choke", "coque", "cob", "legend")
+				var/milk = pick("malk", "milic", "vilk", "mick", "mill", "mink", "mok", "meek", "misk", "music", "dairy")
+				var/thinkpositive = pick("What a great drink.", "Wonder if there's more around?",\
+				"Yum!","You daydream about having more.","Maybe your coworkers want some!","You should get more.")
+				var/thinknegative = pick("The thought sickens you.", "You feel betrayed.",\
+				"You hasten to distract yourself.","You crave vengeance.","Regret washes over you.")
+
+				switch(regret)
+					if(FALSE)
+						if (probmult (6))
+							boutput(M, SPAN_ALERT(pick("Your thoughts turn briefly to [coke] [milk]. [thinkpositive]",\
+							"Your stomach rolls. Must be time for some [coke] [milk]!",\
+							"A delicious taste lingers in your throat!")))
+						else if (probmult(12))
+							if(!M.stat && !M.lying && can_act(M) && !M.equipped() && probmult(20)) //shameless klepto theft
+								for(var/obj/item/reagent_containers/food/drinks/drinkingglass/I in oview(1, M))
+									if(!I.anchored && !I.cant_drop && isturf(I.loc) && can_reach(M, I) && !HAS_ATOM_PROPERTY(I, PROP_MOVABLE_KLEPTO_IGNORE) && I.reagents.has_reagent("legendairy"))
+										I.Attackhand(M)
+										break
+
+							var/obj/item/reagent_containers/food/drinks/helddrink = M.equipped()
+							if(helddrink && helddrink.reagents.has_reagent("legendairy"))
+								helddrink.take_a_drink(M, M) // forces you to drink from held glasses.
+								if (probmult(70)) M.say("Wow! Great drink!") // The iconic line!!
+
+						else if (probmult(6))
+							M.say(pick("[coke] [milk] was great.",\
+								"Mmm. I love [coke] [milk]!",\
+								"I liked that drink!",\
+								"Wow!",\
+								"Gee whillickers, that's refreshing!",\
+								"What legendary taste!",\
+								"Gotta get me a sippy of that dranky drank!",\
+								"This really [coke]s my [milk]!"))
+
+					if(TRUE)
+						if (probmult (6))
+							M.emote("scowl")
+							boutput(M, SPAN_ALERT(pick("Your thoughts turn briefly to [coke] [milk]. [thinknegative]",\
+							"Your stomach rolls. Must be because of that [coke] [milk].",\
+							"The vile taste of [coke] [milk] lingers in your throat.",\
+							"You feel nauseous. You resolve to never drink [coke] [milk] again.")))
+						else if (probmult(6))
+							M.say(pick("I don't want any more of that [coke] [milk].",\
+								"I'm not thirsty anymore.",\
+								"I want to go home.",\
+								"I'm tired.",\
+								"I feel sick.",\
+								"[get_area(M)] looks too exciting for me.",\
+								"I want to be on a station more thrilling than [the_station_name]."))
```

Comment:
Not a fan of drinks that make you say stuff constantly while they're in you (bojaaaacks). Having it make you say something when you compulsively grab a glass and drink from it is funny but the constant "coge milg" spam will get old fast.